Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Paged/WalletSolanaProviderScript.js

```diff
@@ -108,10 +166,10 @@ window.__firefox__.execute(function($, $Object, $Function, $Array) {
           $.extensiveFreeze(object, freezeExceptions);
           return post('signAndSendTransaction', object)
         }),
-        signMessage: $(function(...payload) { /* -> Promise{publicKey: <solanaWeb3.PublicKey>, signature: <Uint8Array>}> */
+        signMessage: $(function(...payload) { /* -> Promise<{publicKey: solanaWeb3.PublicKey, signature: [UInt8]}> */
```

Comment:
We're not changing the return types here accidentally are we?

Reply:
No change here, just cleaned up the comment for consistency with the others.